reana/reana_dev/git.py

```diff
@@ -820,4 +874,45 @@ def _push_to_origin(components):
         _push_to_origin(components)
 
 
+@click.option(
+    "--component",
+    "-c",
+    required=True,
+    multiple=True,
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@click.option(
+    "--next-version", "-v", help="Shall we manually specify component's next version?",
```

Comment:
All works well :+1: One last naming comment: `-v` and `--next-version` do not play well together, it is usually good to keep the first letter identical.  Two options:

- keep `--next-version` name and remove short `-v` variant; we can be simply using long name

- change to `--version` name and keep short `-v`; it makes sense since you are basically saying "git create new release commit with version X.Y.Z"